internal/provider/resource_project.go

```diff
@@ -179,6 +179,7 @@ func (r *ProjectResource) Create(ctx context.Context, req resource.CreateRequest
 		Description: data.Description.ValueString(),
 		IsPublic:    !data.Private.ValueBool(),
 		PrDeploys:   data.HasPrDeploys.ValueBool(),
+		Runtime:     PublicRuntimeV2,
```

Comment:
All the new projects could be created only using V2 runtime. As far as I understood not every existing project is moved to V2 runtime, but new projects can be created only with V2